src/components/pages/events/eventOverview/eventOverview.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect, useState } from 'react';
+import './eventOverview.scss';
+import EventPreview from 'components/molecules/event/eventPreview/EventPreview';
+import useUpcomingEvents from 'hooks/useEvents';
+import LoadingWrapper from 'components/atoms/loadingWrapper/LoadingWrapper';
+import Footer from 'components/molecules/footer/Footer';
+import MyEvents, { DisplayMyEvents } from '../myEvents/MyEvents';
+import { Event } from 'models/apiModels';
+import { getJoinedEvents } from 'api';
+import { sortDate } from 'utils/sorting';
+import { useToast } from 'hooks/useToast';
+
+interface IUpcomingEvents {
+  events: Event[];
+}
+
+export const DisplayUpcomingEventOverview: React.FC<IUpcomingEvents> = ({
+  events,
+}) => {
+  return (
+    <div className="upcomingEventOverview">
+      <div className="upcomingEventOverviewContainer">
+        <div className="events">
+          {events.length ? (
+            events.map((event) => (
+              <EventPreview
+                eventData={event}
+                orientation={'horizontal'}
+                key={event.eid}
+              />
+            ))
+          ) : (
+            <h3 style={{ minHeight: '65vh' }}>Ingen kommende arrangementer </h3>
+          )}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const UpcomingEventOverview: React.FC = () => {
+  const { events } = useUpcomingEvents();
+  return (
+    <>
+      <LoadingWrapper data={events} startAfter={250}>
+        <DisplayUpcomingEventOverview events={events ?? []} />
+      </LoadingWrapper>
+    </>
+  );
+};
+
+type eventContent = 'overview' | 'my events';
+
+const EventOverview: React.FC = () => {
+  const [eventContent, setEventContent] = useState<eventContent>('overview');
+  const { events } = useUpcomingEvents();
+  const [joinedEvents, setJoinedEvents] = useState<Event[] | undefined>();
+  const [joinedErrorMessage, setJoinedErrorMsg] = useState<string>('');
+  const { addToast } = useToast();
+
+  const fetchEvents = async () => {
+    try {
+      /* Get events */
+      const events = await getJoinedEvents();
+      const sorted = [...events].sort((a: Event, b: Event) =>
```

Comment:
Why do we spread on events? Can't we just do `events.sort(...)`?

Reply:
Agree, old habit🙃